meta-xfstk/recipes-xfstk/xfstk/xfstk_1.8.1.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "Build XFSTK Downloader for recovery"
+LICENSE = "LGPL-2.1"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/LGPL-2.1;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit cmake
+
+DEPENDS_${PN} = "g++ qtbase5 qtcreator build-essential devscripts libxml2-dev alien doxygen graphviz libusb-dev libboost-all-dev"
+
+SRC_URI = "https://github.com/Exoyds/xfstk/archive/1.8.2.zip"
```

Comment:
SRC_URI = "git://github.com//Exoyds/xfstk.git;protocol=https"?

Reply:
or https://github.com/saaguirre/xfstk-code?

Reply:
I have set it as an archive as this way any push to the master will not break any builds. Also the repo I have is almost exactly the same as AurelienBailers except for a tiny change in readme. I don't think the saaguirre repo has the QT5 patches, and iirc QT4 was hard to install on 20.04

Reply:
No but they added 16.04 support. 

BTW with a git SRC_URI you need to reference a SRCREV so as long as no force pushes to master still the same rev is checked out.

Reply:
But what I meant is the 16.04 support is just updated library requirements, instead of changing qt4 to qt5. https://github.com/saaguirre/xfstk-code/blob/master/ancillary/configure/api/downloader-api/CMakeLists.txt#L10 for example here it still requires qt4. Also as in my other comment https://github.com/saaguirre/xfstk-code/commit/1bcd3bcf396aa96cb25115b62ac66ac9b9eb5373 that is the changes to update to 16.04 from what I can see.

Do you think it's still worth switching to the other repo?

Also should we switch to following the master branch then instead of releases?

Reply:
1) Ah, yes, I was interested in building the cli, but it links libs not in 20.20, that is a problem. I didn't consider qt.
2) Maybe cherry-pick from one on top of the other?
3) Releases is just a tag in the master branch, no?

Reply:
1. What libs does the cli link in 20.20 that are not present? I built it all in 20.04 including the cli and had all the libs, but if some are missing from 20.20 this could be an issue.
2. I have now pushed the changes as saaguire plus I have added the libs in 20.04 so now we can target 20.04. Also the changes have been tested and it builds just fine.
3. As I've said I'm quite new ! :) I have now pushed the changes as a 1.8.3 release and changed the recipe to track it accordingly.

Reply:
1. With your added libs nothing seems to be missing. I was hoping we could prevent the building of qt by building `make xfstk-dldr-solo` and removing the dependency on qt. But I see now:
```
ferry@delfion:~$ ldd /usr/bin/xfstk-dldr-solo
        linux-vdso.so.1 (0x00007ffcf8dbc000)
        libQt5Core.so.5 => /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 (0x00007f82f07de000)
        libboost_program_options.so.1.71.0 => /usr/lib/x86_64-linux-gnu/libboost_program_options.so.1.71.0 (0x00007f82f074f000)
        libusb-0.1.so.4 => /lib/x86_64-linux-gnu/libusb-0.1.so.4 (0x00007f82f0544000)
        libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f82f0363000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f82f0348000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f82f0156000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f82f0131000)
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f82f0115000)
        libicui18n.so.66 => /usr/lib/x86_64-linux-gnu/libicui18n.so.66 (0x00007f82efe16000)
        libicuuc.so.66 => /usr/lib/x86_64-linux-gnu/libicuuc.so.66 (0x00007f82efc30000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f82efc2a000)
        libpcre2-16.so.0 => /usr/lib/x86_64-linux-gnu/libpcre2-16.so.0 (0x00007f82efba7000)
        libdouble-conversion.so.3 => /usr/lib/x86_64-linux-gnu/libdouble-conversion.so.3 (0x00007f82efb8f000)
        libglib-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007f82efa66000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f82ef917000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f82f0ebe000)
        libicudata.so.66 => /usr/lib/x86_64-linux-gnu/libicudata.so.66 (0x00007f82ede56000)
        libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007f82edde3000)
```